### ISSUE TITLE
blockr.io explorer far behind

### ISSUE THREAD

**Issue by dmsmaxim** [BitGo/bitgo-recovery-tool#2]
Block explorer which uses in this tool (http://blockr.io/) is far behind (about 2 weeks now), so tool doesn't work correctly.

**Comment by jlopp** [comment]
Good point; blockr has been becoming less and less reliable over the years. We should switch to a better API.
